community/kernel/src/main/java/org/neo4j/kernel/impl/factory/CommunityEditionModule.java

```diff
@@ -223,7 +223,13 @@ protected IdGeneratorFactory createIdGeneratorFactory( FileSystemAbstraction fs,
 
     public static Locks createLockManager( Config config, LogService logging )
     {
-        String key = config.get( GraphDatabaseFacadeFactory.Configuration.lock_manager );
+        // If we want to use deferred locking then use the community lock manager since it won't give
```

Comment:
i do not think we should always stick to community lock manager and this comment is misleading